portfolio/src/main/webapp/script.js

```diff
@@ -301,10 +301,10 @@ function buildSectionEntryDescription(description) {
  * @param {!Array<string>} greetings A list of greetings.
  */
 function renderGreetings(greetings) {
-  const mainContainer = document.querySelector('main');
+  const commentList = document.querySelector('main > .comments-section .comments-list');
```

Comment:
when doing a querySelector, try to use the less selectors as possible: this could be `.comments-section .comments-list` or even just `.comments-list`

Reply:
Done!